RecoHI/HiTracking/python/HISelectedTracks_cfi.py

```diff
@@ -65,6 +65,7 @@
     )
 
 #complete sequence
-hiTracksWithQuality = cms.Sequence(hiInitialStepSelector
+hiTracksWithQualityTask = cms.Task(hiInitialStepSelector
                                    #* hiSelectedTracks
 )
+hiTracksWithQuality = cms.Sequence(hiTracksWithQualityTask)
```

Comment:
This sequence was only needed in RecoHI/HiTracking/python/HighPtTracking_PbPb_cff.py, and it can be removed now